perf_test/sparse/CMakeLists.txt

```diff
@@ -103,7 +103,6 @@ KOKKOSKERNELS_ADD_EXECUTABLE(
 KOKKOSKERNELS_ADD_EXECUTABLE(
         sparse_gs
         SOURCES KokkosSparse_gs.cpp
-        TESTONLYLIBS kokkoskernelsperf_gtest
```

Comment:
What's that change about?

Reply:
This linked against the gtest library even though it didn't need to.